### PR TITLE
dm/heroku

### PR DESCRIPTION
**Pull request by svintbel**
Yeap, we are online now  :rocket: :rocket: :rocket:
[staging preview on heroku](https://nicher-events-staging.herokuapp.com/)

Screenshot from 2019-08-06 21-14-46

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: bundle exec rails s
+release: bin/rake db:migrate
```
Comment (LuginaJulia) [inline Procfile]: EOF

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 class WelcomeController < ApplicationController
-  def index; end
+  def index
+    render file: 'welcome/index.html'
```
Comment (M1ha11) [inline app/controllers/welcome_controller.rb]: why do you need this?

Comment (svintbel) [reply app/controllers/welcome_controller.rb]: Without this heroku fails, seems that's why i've added it here.

File: Procfile (hunk 1 above)
Comment (svintbel) [reply Procfile]: done

**Pull request merged**